ksml-runner/src/main/java/io/axual/ksml/runner/lombok.config

```diff
@@ -0,0 +1,3 @@
+lombok.accessors.fluent=true
```

Comment:
Shouldn't we set this as a common resource, and rewrite accordingly, instead of per package?